packages/client/src/types/forms.ts

```diff
@@ -1,22 +1,20 @@
 import { Readable } from "svelte/store"
-import { FieldSchema, FieldType } from "@budibase/types"
+import { FieldSchema, FieldType, UIFieldValidationRule } from "@budibase/types"
 
 export interface FormContext {
   formApi?: {
     registerField: (
       field: string,
       type: FieldType,
-      defaultValue: string | undefined,
+      defaultValue: string | string[] | undefined,
       disabled: boolean,
       readonly: boolean,
-      validation: FieldValidation | undefined,
```

Comment:
Not sure why I set this type. If this does not break any type, I guess it's ok. Checking my typing PR, I don't see the reason why I did that, but just worried that changing from a delegate to an object, nothing complains 😅

Reply:
Yeah, I don't think this is very well hooked up to be honest - I'd like to look at this more, but I have confirmed what is being passed in at the top of the stack and its not a delegate - the structure is based on runtime information.